health-services/project-factory/src/server/templateClasses/user-templateClass.ts

```
@@ -0,0 +1,163 @@
+            [getLocalizedName("HCM_ADMIN_CONSOLE_BOUNDARY_DATA", localizationMap)]: {
+                data: boundaryData,
+                dynamicColumns: boundaryDynamicColumns
+            }
+        }; // Initialize the SheetMap object
+        logger.info(`SheetMap generated for template of type ${responseToSend.type}.`);
+        return sheetMap;
+    }
+
+    static getReadMeData(readMeConfig: any, readMeColumnHeader: any, localizationMap: any) {
+        const dataArray = [];
+        for (const text of readMeConfig?.texts) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)